package.json

```diff
@@ -14,9 +14,9 @@
     "coverage:report": "nyc report --check-coverage --statements 50",
     "lint": "next lint",
     "lint:fix": "next lint --fix",
-    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
-    "prettier:check": "npm run prettier -l",
-    "prettier:fix": "npm run prettier --write"
+    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown",
```

Comment:
This command is meant to just display the prettier errors in the terminal, not fix them. We have `prettier:fix` for that.

Reply:
Pretter:fix is not working, this command also just shows the error like prettier:check does. It is not writing those changes.

Reply:
"prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
    "prettier:check": "npm run prettier -l",
    "prettier:fix": "npm run prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown"

Can I do something like this?

Reply:
"prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
"prettier:check": "npm run prettier -l",
"prettier:fix": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown"

Or this, to be more precise

Reply:
```
"prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
"prettier:check": "npm run prettier -l",
"prettier:fix": "npm run prettier --write"
```

This does not work?

Reply:
No, the prettier:fix: npm run prettier --write  doesn't work.

"prettier:fix": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown". This does.

Reply:
Let's go with [your suggestion](https://github.com/pesto-students/babitz-frontend/pull/19#discussion_r677605891)